src/components/Capture/capture.js

```diff
@@ -252,12 +182,7 @@ class Capture extends Component {
     this.setError('SERVER_ERROR')
   }
 
-  onGlareWarning = () => {
-    this.setWarning('GLARE_DETECTED')
-  }
-
   setError = (name) => this.setState({error: {name, type: 'error'}})
```

Comment:
No, we need to be consistent between backend and frontend errors because if you use webcam for document upload > you click on the upload fallback link > you upload an invalid document > the error will be displayed on the uploader. Actually this edge case should be handled for glare warnings too.